Documentation/markdown/GitProcess.md

```diff
@@ -19,17 +19,33 @@ Before you can proceed to work with Git, there are several steps, which have to
 
 3. *Clone* your repository to local mashine.
 
-	- Open your File Explorer and select the folder, where you want to clone the OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
+	- Open your File Explorer and select the folder, where you want to clone OpenInfraPlatform (e.g. C:\dev). Right mouse click in this folder and select *Git Bash Here*.
 
 	- In the window, which you have opened, in the command line write out *git clone https://github.com/<your_username>/Open-Infra-Platform.git* . This will start a download.
 
-4. Add remote: git add 
+4. Add remote: *git remote add tumcms https://github.com/tumcms/Open-Infra-Platform.git*
 
 ## Submit changes 
 
-branching !!
+1. Before creating any changes, create new branch: 
+ 
+	- Open OpenInfraPlatform folder and select *Git Bash Here*. You also can open **Git Bash** and write out in the command line *cd "<Path to OpenInfraPlatform>"* (e.g  *cd  "C:\dev\Open-Infra-Platform"*)
 
-(Commit + push)
+	- In the command line write out *git checkout -b <branchname>* (e.g *git checkout -b Fixing_Bugs*)
+
+2. After creating some changes, *add* these changes:
+
+	- In the command line of **Git Bash** write out *git add <filename>* to add changes to one file. 
```

Comment:
... to add changes of one file to a commit. Try and be as atomic as possible in commit contents.

Reply:
Corrected